src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.suppliers.FigureSupplier;
+import core.basesyntax.templates.Drawable;
+import core.basesyntax.templates.Figure;

```

Comment:
Do not use plural in package names
```suggestion
import core.basesyntax.supplier.FigureSupplier;
import core.basesyntax.template.Drawable;
import core.basesyntax.template.Figure;

```